### DISCUSSION TITLE
Condition rendering with GetX

### DISCUSSION THREAD

**Discussion by dim0147** [jonataslaw/getx#1734] [Q&A]
Hi, I have created a simple controller like this:
![image](https://user-images.githubusercontent.com/44487221/128460404-a3efa769-0474-4615-9bc7-85bce97a4010.png)
And here is the code in view:
![image](https://user-images.githubusercontent.com/44487221/128460486-dcfb3bfb-1c88-40e8-b5be-3d8959eaf978.png)
But this will output the warning said ` [Get] the improper use of a GetX has been detected.` so I change it like this:
![image](https://user-images.githubusercontent.com/44487221/128460559-29026b20-264f-413d-8a4e-d3d55418e818.png)
But it always show `Text('Loading')` although I have set `isLoadingCategory` equal `false`, how to fix this? Thank

**Comment by jasonlaw** [comment]
Declare you variable as
final isLoadingCategory = true.obs;

In your view use controller.isLoadingCategory() inside the Obx.

**Reply by dim0147** [reply]
Thank you for your reply, if I want to update `isLoadingCategory` what should I do? I currently using `GetBuilder` to solve this

**Reply by dim0147** [reply]
Ah nevermind I was thinking declare `final` mean can't set new value, but forgot it is `Rx<Type>` not `boolean` type

**Reply by jasonlaw** [reply]
Anyway, to update the value, you can either use isLoadingCategory(true) or isLoadingCategory.value = true

**Reply by dim0147** [reply]
Yes thank you for your help
